mappings/net/minecraft/world/spawner/PatrolSpawner.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_3769 net/minecraft/world/spawner/PatrolSpawner
+	COMMENT A spawner for pillager patrols.
```

Comment:
Maybe document that they are spawned around players, so as to distinguish better from the outpost ones?

Reply:
"Pillager patrol" is usually enough IMO.